content/coda.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2022-05-10
+slug: #
```

Comment:
I'm not familiar with how the parsing is handled in this situation. Will the slug value be inserted as an empty string, or does it need to be explicitly defined as `""`?

Reply:
So far as I could tell the slug was for generating links inside the site, so I used the # as I do when I have a link I don't currently want to go anywhere.  When I tested this it did as I expected and created a link just to localhost:1313.  I do not believe the idea is to keep it linked to nothing, but the decision hasn't currently been made where we want this one to link to.

Reply:
Yeah, still deciding where we want it to link to. Let me know if there's no good way to handle this situation and I'll give you somewhere to link it to.

Reply:
there needs to be a string.  The "slug" is the URL that this page would be on

Reply:
@brownd-usgs got a URL for now to use then?